apps/vr-tests-web-components/src/stories/avatar/avatar.stories.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28784&commitId=b6f42873787ebfcedefc43c6bc69f7cdac138eaa&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Active - Dark Mode.normal.chromium.png|0| Added|
|Avatar.Active Appearance - Dark Mode.normal.chromium.png|0| Added|
|Avatar.Active Appearance.normal.chromium.png|0| Added|
|Avatar.Active.normal.chromium.png|0| Added|
|Avatar.Color - Dark Mode.normal.chromium.png|0| Added|
|Avatar.Color - RTL.normal.chromium.png|0| Added|
|Avatar.Color.normal.chromium.png|0| Added|
|Avatar.Colorful - Dark Mode.normal.chromium.png|0| Added|
|Avatar.Colorful - RTL.normal.chromium.png|0| Added|
|Avatar.Colorful.normal.chromium.png|0| Added|
|Avatar.Custom Initials - RTL.normal.chromium.png|0| Added|
|Avatar.Custom Initials.normal.chromium.png|0| Added|
|Avatar.Default - RTL.normal.chromium.png|0| Added|
|Avatar.Default.normal.chromium.png|0| Added|
|Avatar.Icon - Dark Mode.normal.chromium.png|0| Added|
|Avatar.Icon.normal.chromium.png|0| Added|
|Avatar.Image.normal.chromium.png|0| Added|
|Avatar.Shape.normal.chromium.png|0| Added|
|Avatar.Size.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>